src/routes/(app)/import/+page.svelte

```diff
@@ -448,23 +449,23 @@
           thoughts: v.lot === "movie" && v.reviews.length ? v.reviews[0].review.text : "",
 
           // Ryot does not support overall rating for shows
-          rating: v.lot === "movie" && v.reviews.length ? Number(v.reviews[0].rating) : null,
+          rating: v.lot === "movie" && v.reviews.length ? Number(v.reviews[0].rating) : undefined,
```

Comment:
Changed `null` usage to `undefined` to conform to the `rating` property type.

**Irrelevant, but maybe informative if you care:**
I usually try to avoid using `null` and use `undefined` instead. Since JS has null and undefined, they usually tell us the same thing, so I prefer to, in most cases, strictly use `undefined` instead (can't think of any upsides to `null` other than if our logic needs to differentiate between the two for some reason). Anyways that's just what I think.

Reply:
Oh yeah, that makes sense.

JS is not my main language, so I was going with knowledge from a few years ago when I decided on null.
I will keep that in mind for future PRs